index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Polyglot</title>
+    <link rel="icon" href="IMG/logo.jpg" type="image/x-icon">
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+        rel="stylesheet" />
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/ecb44d8fff.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
```

Comment:
# Desktop Size
- **Home Page**: It has a white background and same color.
- See your design
- 
![Capture](https://user-images.githubusercontent.com/63972670/196749075-c40c035e-769d-4f90-a03f-cf719787531a.JPG)
- See the original design
- 
![Capture](https://user-images.githubusercontent.com/63972670/196749399-e09b37d5-84bb-4614-9c39-8ea0f9989a54.JPG)
- **About page** is a dark background. 
- See original design:
- 
![Capture](https://user-images.githubusercontent.com/63972670/196750723-ce900872-5336-4e47-a91f-97d59196150c.JPG)